Package.swift

```diff
@@ -52,10 +52,12 @@ let package = Package(
                 .product(name: "SwiftParser", package: "swift-syntax"),
                 .product(name: "SwiftOperators", package: "swift-syntax"),
                 "SwiftyTextTable",
+                "DyldWarningWorkaround",
```

Comment:
Should this be a conditional dependency so it’s not pulled in on Linux?

Reply:
done